tools/benchmark/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@fluid-tools/benchmark",
-	"version": "0.50.0",
+	"version": "0.51.1",
```

Comment:
Why 0.51.1? If this is a breaking API change, it should go to 51.0, otherwise 50.1.